Bottom: 081ae8272ed9b21443434c4789553e0debb4c3dc
Top:    ca0891a2a16862ef8306ab3362aed3554d274d68
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-22 17:16:44 +0800



---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 37e9015..fdd5bcc 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -755,6 +755,7 @@ static void handle_rx(struct vhost_net *net)
 				vhost_len = vq->heads[0].len =
 					cpu_to_vhost32(vq,
 						iov_length(vq->iov, in));
+				printk("len is %d\n", vq->heads[0].len);
 				sock_len = vhost_len - vhost_hlen;
 				headcount = 1;
 				cur++;
@@ -829,6 +830,7 @@ static void handle_rx(struct vhost_net *net)
 			vhost_discard_vq_desc(vq, headcount);
 			goto out;
 		}
+		printk("add %d\n", vq->heads[0].len);
 		vhost_add_used_and_signal_n(&net->dev, vq, vq->heads,
 					    headcount);
 		if (unlikely(vq_log))
